Libplanet/Blockchain/Renderers/IActionRenderer.cs

```diff
@@ -30,6 +32,15 @@ namespace Libplanet.Blockchain.Renderers
     /// (one time)</description></item>
     /// </list>
     /// </summary>
+    /// <remarks>Although <see cref="Transaction{T}"/>s affect the states in the <see
+    /// cref="IStore"/> all or nothing at all (i.e., atomically), <see cref="IActionRenderer{T}"/>
```

Comment:
It has the same context with the review comment in `CHANGES.md`. Does the `states` mean state in `IStateStore`? If then, it seems better to use `IStateStore` instead of `IStore`.

Reply:
Amended: 1f1ed497c892b56c6802a9d15cdeb4cddd1b98db.